helix-core/src/syntax/span.rs

```diff
@@ -17,9 +21,9 @@ pub struct Span {
 
 impl Ord for Span {
     fn cmp(&self, other: &Self) -> std::cmp::Ordering {
-        // Sort by range: ascending by start and then ascending by end for ties.
+        // Sort by range: ascending by start and then descending by end for ties.
         if self.start == other.start {
-            self.end.cmp(&other.end)
+            other.end.cmp(&self.end)
```

Comment:
minor nit but I prefer `self.end.cmp(&other.end).reverse()` since it's more readable IMO

Reply:
fixed in 20302313b5d274d231c8c2e505a5306601c0173d